### ISSUE TITLE
API 6.2 beta: Deleting podcast -> 405

### ISSUE THREAD

**Issue by Whisprin** [Libresonic/libresonic#307]
Hello,

trying to delete a podcast from Libresonic, as admin or user, results in an error. 
edit: It seems that many podcast related operations are not available:
- Delete podcast
- Download selected episode(s)
- Delete selected episode(s)
- Check for new episodes

The error log for delete is attached.
![libresonic-delete-podcast](https://cloud.githubusercontent.com/assets/3153136/24113529/5da7f1fc-0d9d-11e7-91c3-2d0dbc1ff311.png)

[libresonic-delete-podcast.txt](https://github.com/Libresonic/libresonic/files/855857/libresonic-delete-podcast.txt)

Steps to reproduce:
1. Open "Podcasts" in Libresonic
2. Add a podcast by entering it's URL, e.g.: https://alternativlos.org/opus.rss
3. Open that podcast, press on delete and confirm the deletion

It seems that the Libresonic webinterface is doing a GET instead of POST or DELETE towards the API endpoint of the server.

**Comment by muff1nman** [comment]
Can you send a screenshot of your about page so we can double check the version? IIRC there are some changes on the develop branch for something similar.

**Comment by muff1nman** [comment]
Might be a duplicate of #296

**Comment by Whisprin** [comment]
Here's my about page:
![screenshot from 2017-03-21 10-42-12](https://cloud.githubusercontent.com/assets/3153136/24141218/17d281de-0e23-11e7-8bc7-9a04460c6156.png)

It seems that #296 could fix all of the mentioned problems. But I couldn't verify it yet because my war build refuses to start.

**Comment by muff1nman** [comment]
I have noticed this sometimes too so created a bug for it #311. I would just make sure that tomcat is completely killed before starting a new instance.

**Comment by muff1nman** [comment]
Closing this as its definitely a dup of #296.
